app.js

```diff
@@ -19,9 +18,20 @@ var app = express();
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'ejs');
 
+// Authentication 
+app.use(session({
+  secret: 'secret',
+  resave: true,
+  saveUninitialized: false,
+  cookie: { maxAge: 60000, secure: false }
```

Comment:
Need to include httpOnly: true for one more layer of security.

Reply:
wait httpOnly? or httpsOnly?

Reply:
ok added this